src/schemas/skill.schema.ts

```diff
@@ -0,0 +1,10 @@
+  
+import { object, string, ref } from 'yup';
+
+export const createSkillSchema = object({
+  body: object({
+    label: string()
+    .required(`Label is required`)
+    .matches(/^[a-zA-Z0-9_.-]*$/, `Label can only contain Latin letters.`),
```

Comment:
In my opinion, you can change this reggex by a new reggew with allow only letters and not numbers.